tests/tester_pframework_selector.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from unittest.mock import patch
+from pframework import TrafficPredictor
+
+
+class TesterPframeworkSelector(unittest.TestCase):
+    predictor = TrafficPredictor()
+
+    @patch('builtins.input')
+    def test_selecting_model_test(self, mocked_input):
+        try:
+            mocked_input.side_effect = ['0']
+            self.predictor.main()
+        except Exception as error:
+            self.fail("Test: Failed - {0}\n".format(error))
+
+    def test_importing_model_test(self):
+        try:
+            result = self.predictor.call_model('model_test')
+            self.assertEqual(result, "This is model test!")
+        except Exception as error:
+            self.fail("Test: Failed - {0}\n".format(error))
+
+    def test_importing_wrong_model(self):
```

Comment:
https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception

Maybe try something like that here.

Reply:
That's a nice one! Gonna try this out tomorrow.

Reply:
Actually, for this test to pass, it means that it does not raise an exception on line 26. Is that correct? call_model should throw an exception when an invalid model name is chosen.

Reply:
Well, I'm not sure anymore if we can utilize this because I haven't found a good way to raise error. Because it's going to display a stacktrace, I don't think it will be a good way to display the error if it will happen in prod.

Reply:
I don't think that's a problem. You could have that try catch block around line 35 in main instead of writing it in the function itself.